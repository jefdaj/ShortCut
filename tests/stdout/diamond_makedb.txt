DIAMOND database $TMPDIR/vars/result

