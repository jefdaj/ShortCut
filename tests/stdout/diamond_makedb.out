DIAMOND database $TMPDIR/vars/result
Database format version = 2
Diamond build = 123
Sequences = 194
Letters = 69516

